.clusterfuzzlite/README.md

```diff
@@ -0,0 +1,4 @@
+# ClusterFuzzLite set up
+
+This folder contains a fuzzing set for [ClusterFuzzLite](https://google.github.io/clusterfuzzlite).
```

Comment:
Please add the instructions you wrote on how to run it locally.

Are these instructions deterministic? I am a bit afraid that a PR is red because of a failing test and locally the failure cannot be reproduced. Even worse if a red PR can be made green by rerunning the tests...

Reply:
Done.

It's deterministic in the sense that these instructions will always run the fuzzer. It's non-deterministic in the sense that there is an element of randomness in fuzzing, so running these steps twice will not produce the exact same fuzzing run.

If the fuzzer finds an issue then ClusterFuzzLite will save the input that causes the crash and make it available for download, this can then be used to reproduce the issue in a determinist way.